Tuist/ProjectDescriptionHelpers/Configurations.swift

```diff
@@ -0,0 +1,12 @@
+import ProjectDescription
+
+extension Array<Configuration> {
+
+    /// Debug and Release configurations that use the Global.xcconfig file
+    public static var xcconfigFileConfigurations: [Configuration] {
+        [
+            .debug(name: "Debug", xcconfig: .relativeToRoot("Global.xcconfig")),
+            .release(name: "Release", xcconfig: .relativeToRoot("Global.xcconfig")),
```

Comment:
I am not sure about this, I think it shouldn't point to the `Global.xcconfig` right @ajpallares ?